task/api/README.md

```diff
@@ -37,47 +46,67 @@ found or even expired password.
 
 #### Composite Node
 Composite node defines a list of subgraphs to evaluate. It enables `parallel` processing of independent 
-nodes/subgraphs (e.g. calling two external independent data sources).
+subgraphs (e.g. calling two external independent data sources). Each subgraph gets a fragment copy and 
+updates a fragment's payload. When all subgraphs end, then the reduction phase happens during which 
+all subgraphs results are merged (all copies' payloads).
```

Comment:
So are we calling them subgraphs or subtasks? The word `subtask` was already introduced and now it's not used

Reply:
Replaced all `subgraphs` with `subtasks`.